src/server/saved_objects/service/lib/index.js

```diff
@@ -19,6 +19,7 @@
 
 export { SavedObjectsRepository } from './repository';
```

Comment:
Do we still want to expose `SavedObjectsRepository` here?

Reply:
I think it makes sense here since it is a module that we want to generally expose from the `lib` directory. That's at least how I understand the [style guide](https://github.com/elastic/kibana/blob/master/style_guides/architecture_style_guide.md#subdirectories), but I'm open to reinterpreting this!

Reply:
When would someone use the SavedObjectsRepository as opposed to the SavedObjectsRepositoryProvider?

Reply:
In general: if someone needs more flexibility than the provider allows, having direct access to the Repository will let them construct an instance that meets their specific needs. The provider only allows `callCluster` to be customized, but there are other parameters that callers may want to adjust